### PR TITLE
Update wbebhook suppression information

### PR DESCRIPTION
**Pull request by onlyhavecans**
Also vscode web autofixed broken lints on this document. I could revert them if desired.

However I generally am ok with lintfixes.

### CODE REVIEW

File: content/v2/webhooks/events.markdown [hunk 1]
```diff
@@ -158,9 +157,6 @@ If your webhook endpoint has been disabled no further retry attempts are made fo
 If your webhook endpoint has been misconfigured and has not responded with a 200 HTTP status code consistently for up to 15 attempts DNSimple will automatically suppress the webhook endpoint.
 
 <note>
-If you think your webhook has been disabled after you stop receiving events. There are two options to remove re-enable the webhook.
-
-1. You can delete and then re-create the webhook in your account.
-
-2. Reach out to [DNSimple support](https://dnsimple.com/contact).
+You can view the health of your webhook and remove suppressions from your account page.
+Please see the [support page on webhooks](https://support.dnsimple.com/articles/webhooks/) for more information.
```
Comment (dallasread) [inline content/v2/webhooks/events.markdown:161]: WDYT about linking to the suppressions section?

```suggestion
Please see the [support page on webhooks](https://support.dnsimple.com/articles/webhooks/#clearing-suppression) for more information.
```

**Review by dallasread — APPROVED** [review]
Thanks for fixing the wbebhooks article!

It'd be nice to do a linting check via a GH Action, but no strong objection to this update. I didn't know `*` was preferable. 🤷‍♂️

**Comment by onlyhavecans** [comment]
> It'd be nice to do a linting check via a GH Action

There is an action that does this, and we use it elsewhere `nosborn/github-action-markdown-cli@v2.0.0`

**Review by DXTimer — APPROVED** [review]
Thank you for fixing this.

**Review by itsalyse — APPROVED** [review]

**Pull request merged**